### PR TITLE
Temp docs

### LINKED ISSUE #1486: Document TEMPORARY behaviour in RediSearch 2.0
needs documenting that it will delete documents 
Please use warning symbol?

### LINKED ISSUE #1500: Document 'startswith' function

### PR DESCRIPTION
**Pull request by ashtul**
* test `!term`
* add doc about temp index not deleting hashes
* add example from blog answer
fix #1486 
fix #1500 

https://squidfunk.github.io/mkdocs-material/reference/admonitions/

### CODE REVIEW

**Comment by ashtul** [comment]
@K-Jo 
Many of the changes are due to the fact `!!! note` or `!!! warning` was not presented correctly.
https://squidfunk.github.io/mkdocs-material/reference/admonitions/

File: docs/Commands.md [hunk 1]
```diff
@@ -57,17 +54,19 @@ FT.CREATE idx ON HASH PREFIX 1 doc: SCHEMA name TEXT SORTABLE age NUMERIC SORTAB
...
+    The supported languages are:
 
     > "arabic",  "danish",    "dutch",   "english",   "finnish",    "french",
     > "german",  "hungarian", "italian", "norwegian", "portuguese", "romanian",
     > "russian", "spanish",   "swedish", "tamil",     "turkish"
     > "chinese"
 
-  If indexing a Chinese language document, you must set the language to `chinese`
-  in order for Chinese characters to be tokenized properly.
+    If indexing a Chinese language document, you must set the language to `chinese`
```
Comment (ashtul) [inline docs/Commands.md:68]: These spaces are so it is still included in the above warning box

File: docs/Commands.md [hunk 2]
```diff
@@ -188,12 +190,13 @@ If a field fails to be indexed (for example, if a numeric fields gets a string v
...
+If `LANGUAGE_FIELD`, `SCORE_FIELD`, or `PAYLOAD_FIELD` were used with `FT.CREATE`, the document will extract the properties. A field can be used to get the name of the index it belongs to.
 
-Complete list of redis commands which might modify the index:
-  HSET, HMSET, HSETNX, HINCRBY, HINCRBYFLOAT, HDEL, DEL, SET, RENAME_FROM, RENAME_TO, TRIMMED, RESTORE, EXPIRED, EVICTED, CHANGE, LOADED
+!!! warning "Schema mismatch"
+    If a value in a hash does not match the schema type for that field, indexing of the hash will fail. The number of 'failed' document is under `hash_indexing_failures` at `FT.INFO`.
 
-If `LANGUAGE_FIELD`, `SCORE_FIELD`, or `PAYLOAD_FIELD` were used with `FT.CREATE`, the document will extract the properties. A field can be used to get the name of the index it belongs to.
+!!! info "Complete list of redis commands which might modify the index:"
+    HSET, HMSET, HSETNX, HINCRBY, HINCRBYFLOAT, HDEL, DEL, SET, RENAME_FROM, RENAME_TO, TRIMMED, RESTORE, EXPIRED, EVICTED, CHANGE, LOADED
```
Comment (ashtul) [inline docs/Commands.md:199]: again boxes and shifting around paragraphs

File: src/pytest/test_followhashes.py [hunk 3]
```diff
@@ -596,3 +597,15 @@ def testDocIndexedInTwoIndexes():
     env.expect('FT.SEARCH idx1 foo').equal([0L])
 
     env.expect('FT.DROPINDEX idx1 DD').ok()
+
+def testCountry(env):
```
Comment (ashtul) [inline src/pytest/test_followhashes.py:601]: checking other functions on `ft.create` `filter`

File: src/pytest/test_issues.py [hunk 4]
```diff
@@ -19,3 +21,9 @@ def test_1414(env):
   env.expect('FT.CONFIG set MAXSEARCHRESULTS -1').equal('OK')
   env.expect('ft.search idx * limit 0 1234567').equal([1L, 'doc', ['foo', 'hello', 'bar', 'world']]) 
   
+def testNotOnly(env):
```
Comment (rafie) [inline src/pytest/test_issues.py:24]: Why in `test_issues.py`?

Comment (ashtul) [reply src/pytest/test_issues.py]: Moved into `test.py`

**Review by rafie — APPROVED** [review]

**Pull request merged**